src/motor_node.cc

```diff
@@ -47,6 +47,9 @@ static FirmwareParams g_firmware_params;
 static CommsParams    g_serial_params;
 static NodeParams     g_node_params;
 
+// TODO: Enhancement - Make WHEEL_SLIP_THRESHOLD be a ROS param
+#define WHEEL_SLIP_THRESHOLD  (0.08)   // Rotation below which we null excess wheel torque in 4wd drive_type
```

Comment:
Wouldn't this be more like motor stall threshold? Also affects heavily loaded 2wd robots and could prevent the occasional annoying buzz I'd imagine?